SPECS-EXTENDED/mbedtls/mbedtls.spec

```
@@ -0,0 +1,300 @@
+Summary:        Light-weight cryptographic and SSL/TLS library
+Name:           mbedtls
+Version:        2.28.3
+Release:        1%{?dist}
+Summary:        Light-weight cryptographic and SSL/TLS library
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.trustedfirmware.org/projects/mbed-tls
+Source:         https://github.com/Mbed-TLS/%{name}/archive/refs/tags/v%{version}.tar.gz
```

Lets rename the source as `%{name}-%{version}.tar.gz`. This makes life easy with automation,
```suggestion
Source:         https://github.com/Mbed-TLS/%{name}/archive/refs/tags/v%{version}.tar.gz#%{name}-%{version}.tar.gz
```

---

@mfrw But the source file format is just v2.28.3.tar.gz, so will it not create a problem.

---

Sure, it will; so we will rename the source as well while we upload to blobstore.
[In this case, since you do not have the permissions to upload, I will upload on your behalf; I will rename the source]
Reference: [last para of this page](https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/633/update_cgmanifest.py)

---

Okay, got it.

---

done.

---

Probably not :)
can you please take a look again at the suggestion.

---

Updated as suggested.